roles/openshift_health_checker/openshift_checks/etcd_volume_check.py

```diff
@@ -0,0 +1,49 @@
+# pylint: disable=missing-docstring
+"""
+Ansible module for determining information about etcd volume size.
+"""
+
+from openshift_checks import OpenShiftCheck, get_var
+
+
+class EtcdVolumeCheck(OpenShiftCheck):
+    """Check that etcd volume capacity is not over 50%"""
+
+    name = "etcd_volume_check"
+    tags = ["etcd_volume_check", "etcd"]
+
+    def run(self, tmp, task_vars):
+        ansible_mounts = get_var(task_vars, "ansible_mounts")
+        etcd_mountpath = self.get_etcd_mountpath(ansible_mounts)
+        etcd_diskspace_avail = etcd_mountpath.get("size_available", 0)
+        etcd_totaldiskspace = etcd_mountpath.get("size_total", 0)
+
+        # max disk usage limit
+        diskspace_usage_lim = get_var(task_vars, "etcd_volume_usage_limit_bytes")
```

Comment:
Thanks, I was not entirely sure how to address this, but for now, I have defaulted this value to 90% of the total disk space